architect.js

```diff
@@ -410,6 +410,7 @@ function Architect(config) {
     var services = app.services = {
         hub: {
```

Comment:
If we are allowing  plugins to access app, using imports, would  it be better to do `hub: app` instead?

Reply:
some plugins may already be called "app" and would confuse, and as for "hub" is already identified as a core architect plugin that "looks" to be barely used and could be used for "architect app" events for plugins, example plugin would be a testing plugin, using hub events to check plugins for test, and then could be removed without modifying plugin config files